specification.swagger.yaml

```diff
@@ -965,57 +694,762 @@ paths:
+          description: Internal id of street. Notice, not persistent. Should only be used
+            when resolving street ids from address resources.
+          schema:
+            type: integer
+          example: 2
       responses:
-        200:
+        "200":
           description: A single street resource
-          schema:
-            $ref: '#/definitions/street'
-
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/street"
   /address/:
     get:
       operationId: Retrieve a list of addresses
       summary: Retrieve a list of addresses
       parameters:
         - name: lat
           in: query
-          type: string
           description: The latitude coordinate to be used in reverse geocoding
+          schema:
+            type: number
+          example: 20.20
         - name: lon
           in: query
-          type: string
           description: The longitude coordinate to be used in reverse geocoding
+          schema:
+            type: number
+          example: 20.20
         - name: language
           in: query
-          type: string
-          description: The two-character ISO 639-1 language code to be used in the street names of the addresses. If missing, the street name parameter will use the default language Finnish.
+          description: The two-character ISO 639-1 language code to be used in the street
+            names of the addresses. If missing, the street name parameter will
+            use the default language Finnish.
+          schema:
+            type: string
+          example: fi
         - name: municipality
           in: query
-          type: string
-          description: A municipality id. The returned addresses will be located within the municipalitie\'s boundaries. The municipality id is either a municipality name in Finnish, or a full OCD ID of the form ocd-division/country\:fi/kunta\:helsinki (URL encoded) See the [OpenCivicData Site](http://opencivicdata.readthedocs.io/en/latest/ocdids.html) for more information
+          description: A municipality id. The returned addresses will be located within the
+            municipalitie\'s boundaries. The municipality id is either a
+            municipality name in Finnish, or a full OCD ID of the form
+            ocd-division/country\:fi/kunta\:helsinki (URL encoded) See the
+            [OpenCivicData
+            Site](http://opencivicdata.readthedocs.io/en/latest/ocdids.html) for
+            more information
+          schema:
+            type: string
+          example: espoo,vantaa
         - name: municipality_name
           in: query
-          type: string
-          description: A municipality name to be used as a filter, in the language specified by the language parameter.
+          description: A municipality name to be used as a filter, in the language
```

Comment:
I note, that this overlaps with municipality-parameter, only that this supports specifying the name in multiple languages.

Reply:
Yes. This is there to support user-entered values for municipalities in UIs.

Reply:
So should it be in swagger?

Reply:
Certainly, I just noted that same functionality is provided by two parameters. Perhaps express a preference here?